src/main/java/uk/gov/justice/record/link/controller/UserTransferController.java

```diff
@@ -84,14 +85,18 @@ public String userLinked(@Validated(SubmissionValidationSequence.class) @ModelAt
         final List<String> expectedErrorMessages = List.of(ValidationConstants.INVALID_LOGIN_ID_MESSAGE,
                                                            ValidationConstants.INVALID_STATUS_MESSAGE,
                                                            ValidationConstants.CCMS_ACCOUNT_CLOSED,
-                                                           ValidationConstants.INVALID_FIRM_ID_MESSAGE);
+                                                           ValidationConstants.INVALID_FIRM_ID_MESSAGE,
+                                                           ValidationConstants.ACCOUNT_ALREADY_ASSIGNED);
 
         final List<String> errors = result.getAllErrors().stream().map(ObjectError::getDefaultMessage)
                 .filter(expectedErrorMessages::contains).toList();
 
+        final Optional<String> accountAlreadyExistError = getAccountAlreadyAssignedError(result);
+
         if (!errors.isEmpty()) {
             log.error("Invalid user transfer request with login id: {}", userTransferRequest.getOldLogin());
             userTransferService.rejectRequest(userTransferRequest, errors.getFirst());
+            accountAlreadyExistError.ifPresent(s -> model.addAttribute("userAssignedError", s));
```

Comment:
Can we not pass the error message here and check what error message is in the HTML file